package.json

```diff
@@ -6,8 +6,7 @@
   "types": "dist/index.d.ts",
   "scripts": {
     "build": "tsc",
-    "lint": "tslint --project tsconfig.json",
-    "test": "tslint --project tsconfig.json"
+    "lint": "tslint --project tsconfig.json"
```

Comment:
should this be eslint?

Reply:
Not yet. This is add in this PR: https://github.com/libero/event-types/pull/6